### PR TITLE
BTA-1479 : CT - View statement link

### PR DESCRIPTION
**Pull request by nehakannaujia1**
BTA-1479 : CT - View statement link

### CODE REVIEW

File: app/views/partials/card/payments/in_credit.scala.html [hunk 1]
```diff
@@ -20,9 +20,22 @@
 
 @(amount: BigDecimal, appConfig: FrontendAppConfig)(implicit request: AuthenticatedRequest[_], messages: Messages)
 
+@statementLink = @{
+ Some(appConfig.getPortalUrl("balance")(request.ctEnrolment)).map { url => url }
```
Comment (suyangyu) [inline app/views/partials/card/payments/in_credit.scala.html]: I don't see why this needs to be lifted into a some?
this applies to the other templates

File: app/views/partials/card/payments/in_credit.scala.html [hunk 2]
```diff
@@ -20,9 +20,22 @@
...
+  @messages("card.view_your_corporation_tax_statement")
+ </a>
+</p>
+
+<p>
+ <a id="make-ct-payment" href="@appConfig.getUrl("mainPage")/make-a-payment" data-sso="false"
+   data-journey-click="link - click:CT cards:Make a CT payment">
+   @messages("card.make_a_corporation_tax_payment")
+ </a>
+</p>
```
Comment (suyangyu) [inline app/views/partials/card/payments/in_credit.scala.html]:
```suggestion
</p>

```

File: test/views/partials/card/payments/InCreditViewSpec.scala [hunk 3]
```diff
@@ -44,6 +44,12 @@ class InCreditViewSpec extends ViewSpecBase {
       doc.getElementsByTag("p").first().text() mustBe "You are £123.45 in credit."
     }
 
+    "must have 'View your Corporation Tax statement' link" in {
+      doc.getElementById("view-ct-statement").text() mustBe "View your Corporation Tax statement"
+      doc.getElementById("view-ct-statement").attr("href", "http://localhost:8080/portal/corporation-tax/org/utr/account/balanceperiods?lang=eng")
```
Comment (suyangyu) [inline test/views/partials/card/payments/InCreditViewSpec.scala]: this is not a test condition, attr actually add/updates the attribute to the element.

this applies in other places as well

File: app/views/partials/card/payments/in_credit.scala.html (hunk 1 above)
Comment (nehakannaujia1) [reply app/views/partials/card/payments/in_credit.scala.html]: Done

File: test/views/partials/card/payments/InCreditViewSpec.scala (hunk 3 above)
Comment (nehakannaujia1) [reply test/views/partials/card/payments/InCreditViewSpec.scala]: Done.

**Pull request merged**